tmt/base.py

```diff
@@ -3113,12 +3114,12 @@ def init(
                     path = tree.root
                 except fmf.utils.GeneralError as error:
                     raise tmt.utils.GeneralError(
-                        f"Failed to initialize tree in '{path}': {error}")
-                echo(f"Tree '{tree.root}' initialized.")
+                        f"Failed to create '{fmf_dir}': {error}")
```

Comment:
Can you also drop `error` from the message, and replace it with `raise … from error`? We are slowly moving to chaining exceptions and since you are changing the line, it’s a good opportunity.

Reply:
Done. Let's see how it works.